WireDomain/Sources/WireDomain/Repositories/Conversations/ConversationLocalStore+MLS.swift

```diff
@@ -19,11 +19,683 @@
 import WireAPI
 import WireDataModel
 
-/// An extension that encapsulates storage operations related to conversation MLS.
+/// An extension that encapsulates storage operations related to MLS.
 
 extension ConversationLocalStore {
+    
+    // MARK: - Public
+    
+    public func addMLSMessage(
+        _ encryptedMessage: String,
+        mlsGroupID: MLSGroupID,
+        mlsConversation: ZMConversation,
+        senderID: UUID,
+        senderDomain: String,
+        subconversation: String?,
+        date: Date?
+    ) async {
+        
+        let decryptionResults = await decryptMLSMessage(
+            message: encryptedMessage,
+            mlsGroupID: mlsGroupID,
+            subconversation: subconversation
+        )
```

Comment:
First we decrypt the the MLS message using the MLSDecryptionService (see `EventDecoder+MLS` legacy code)